setup.py

```diff
@@ -0,0 +1,12 @@
+from distutils.core import setup, find_packages
```

Comment:
`find_packages` is unused, which brings me to: can we get some linting on this project from the start, rather than having to retrofit it later?